models/Headers.yaml

```diff
@@ -0,0 +1,28 @@
+openapi: 3.1.0
```

Comment:
openapi: 3.1.0??? Are we moving to 3.1 already

Reply:
Just for the headers file, 3.0.3 does not allow empty path.